Dockerfile

```diff
@@ -1,51 +1 @@
-# We use the iqsharp-base image, as that includes
```

Comment:
For use on Binder in practice, we should note updates to the katas image won't be picked up without a corresponding commit in this repo. Adding an action that makes that commit would be a good addition after bringing this PR in.

Reply:
Yes, we will need to do a commit anyway to point to the release version/tag.